operations/power_management/Power_On_Compute_Cabinets.md

```diff
@@ -41,6 +41,21 @@ power-on command from Cray System Management \(CSM\) software.
 
    ![PSU Status LEDs](../../img/operations/PSU_Status.svg)
 
+1. (`ncn-m001#`) Unsuspend the `hms-discovery cronjob` to re-enable the `hms-discovery` job.
```

Comment:
Redundant and kind of incorrect, IMO. Suggestion:

```suggestion
1. (`ncn-m001#`) Unsuspend the `hms-discovery` cronjob:
```

Reply:
fixed